### DISCUSSION TITLE
[blocks]: expandable sidebar

### DISCUSSION THREAD

**Discussion by monstajoe2002** [shadcn-ui/ui#4177] [Blocks Request]
### Description

I would like to implement a sidebar that opens and closes like [this website](https://t3-app-template.vercel.app/), but for SvelteKit

### Example

https://t3-app-template.vercel.app/

**Comment by sizuki159** [comment]
Can I ask where you found this website?

**Reply by monstajoe2002** [reply]
On Google

**Reply by sakibhasandev** [reply]
@sizuki159 https://github.com/gaofubin/t3-app-template
